client/src/app/tabs/cloud-bpmn/linting/BpmnLinter.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { Linter } from 'bpmnlint';
+
+import BpmnModdle from 'bpmn-moddle';
+
+import { isString } from 'min-dash';
+
+import linterConfig from '../../.bpmnlintrc';
+
+const moddle = new BpmnModdle();
```

Comment:
Shouldn't we instantiate moddle with `camunda` and `zeebe` namespace awareness?

Alternative: Don't lint files for the moment (throw a `not supported` error).

Reply:
Fixed (cf. https://github.com/camunda/camunda-modeler/pull/2610/commits/2ee2552494cb2fb79a523b5a5a0314f032e9ebab).